docs/src/tutorials/creating_geometry.md

```diff
@@ -0,0 +1,303 @@
+# Creating Geometry
+
+````@example creating_geometry
+import GeoInterface as GI
+import GeometryOps as GO
+import GeoFormatTypes as GFT
+import CoordinateTransformations
+import Proj
+using CairoMakie
+using GeoMakie
+using GeoJSON
+Makie.set_theme!(Makie.MAKIE_DEFAULT_THEME) # hide
+````
+
+The first thing we need to do is decide which [Coordinate Reference System (CRS)](https://en.wikipedia.org/wiki/Spatial_reference_system) we will be working in. Here, we start with the most common geographic CRS (i.e. coordinates of latitude and longitude), [WGS84](https://epsg.io/4326).
+
+````@example creating_geometry
+crs = GFT.EPSG(4326)
+````
+
+Let's start by making a single `Point`.
+
+````@example creating_geometry
+point = GI.Point(0, 0)
+````
+
+Now, let's plot our point.
+
+````@example creating_geometry
+fig, ax, plt = plot(point)
+````
+
+Let's create a set of points, and have a bit more fun with plotting.
+
+````@example creating_geometry
+x = [-5, 0, 5, 0]
+y = [0, -5, 0, 5]
+points = GI.Point.(zip(x,y))
+plot!(ax, points; marker = '✈', markersize = 30)
+fig
+````
+
+Points can be combined into a single `MultiPoint` geometry. This time let's include information on CRS with the geometry making it a geospatial geometry. All that's needed is to include `; crs = crs` as a keyword argument when constucting the geometry.
+
+!!! note
+    It is good practice to only include CRS information with the highest-level geometry. Not doing so can bloat the memory footprint of the geometry. CRS information _can_ be included at the individual `Point` level but is discouraged.
+
+````@example creating_geometry
+x = [-5, -5, 5, 5]
+y = [-5, 5, 5, -5]
+multipoint = GI.MultiPoint(GI.Point.(zip(x, y)); crs)
+plot!(ax, multipoint; marker = '☁', markersize = 30)
+fig
+````
+
+Let's create a line between two points.
+
+````@example creating_geometry
+p1 = GI.Point.(-5, 0)
+p2 = GI.Point.(5, 0)
+line = GI.LineString([p1,p2]; crs)
+plot!(ax, line)
+fig
+````
+
+Now, let's create a line connecting multiple points (i.e. a `LineString`).
+This time we get a bit more fancy with point creation.
+
+````@example creating_geometry
+r = 2;
+k = 10;
+ϴ = 0:0.01:2pi
+x = r .* (k + 1) .* cos.(ϴ) .- r .* cos.((k + 1) .* ϴ)
+y = r .* (k + 1) .* sin.(ϴ) .- r .* sin.((k + 1) .* ϴ)
+lines = GI.LineString(GI.Point.(zip(x,y)); crs)
+plot!(ax, lines; linewidth = 5)
+fig
+````
+
+We can also create a single `LinearRing` trait, the building block of a polygon.
+A `LinearRing` is simply a `LineString` with the same beginning and endpoint,
+i.e., an arbitrary closed shape composed of point pairs.
+
+A `LinearRing` is composed of a series of points listed in clockwise order (i.e., winding order).
```

Comment:
That's true, but I suspect polygons will need the winding order.  Maybe we move the explanation to the polygon section instead?

Reply:
Where will they need it though?

Reply:
https://gis.stackexchange.com/questions/119150/order-of-polygon-vertices-in-general-gis-clockwise-or-counterclockwise

Reply:
There is nowhere in GeoInterface that fixes the winding order of shapefiles currently. Maybe it should. And some other formats allow both ways.

Reply:
Right but do our algorithms actually work with arbitrary winding order?

Reply:
I havent written anything winding dependent. @skygering have you?

I guess we need to extend the tests to generate Shapefile.jl objects with the reveresed windings.

Reply:
The signed area algorithm might be winding dependent, I have to check.  I remember some issues with the wrong winding order though.

Reply:
I have tried to write them so they work with arbitrary winding order. Even if I didn't, I think we have to since we accept polygons with arbitrary winding order. If not, then we need to add another "fix" function that changes the winding order. But overall, I agree with @rafaqz

Reply:
This needs to be carefully traced... winding order has been a pain in my but for years.. if JuliaGeo solves this than all the better but if we don't track carefully it could introduce bugs and compatibility issues... I believe shapefiles use  winding order. For this tutorial I've removed discussion.